src/components/articles/Article.jsx

```diff
@@ -17,58 +25,94 @@ export const Article = () => {
 
     Promise.all(promiseArr).then((data) => {
       setArticle(data[0].article);
-      setFakeKudos(data[0].article.votes);
       setCommentList(data[1].comments);
+      setFakeKudos(data[0].article.votes);
+      setIsArticleLoading(false);
+      setIsCommentsLoading(false);
     });
-  }, []);
+  }, [article_id]);
 
   const handelThumbUp = () => {
     setFakeKudos(fakeKudos + 1);
-    patchVotes(article_id, 1)
-      .then((data) => {})
-      .catch((err) => {
-        setFakeKudos(fakeKudos - 1);
-      });
+    patchVotes(article_id, 1).catch((err) => {
+      setFakeKudos(fakeKudos - 1);
+    });
   };
-  
+
   const handelThumbDown = () => {
     setFakeKudos(fakeKudos - 1);
-    patchVotes(article_id, -1)
-      .then((data) => {})
-      .catch((err) => {
-        setFakeKudos(fakeKudos + 1);
-      });
+    patchVotes(article_id, -1).catch((err) => {
+      setFakeKudos(fakeKudos + 1);
+    });
+  };
+
+  const handleCommentChange = (e) => {
+    setNewComment(e.target.value);
+  };
+
+  const handleCommentSubmit = (e) => {
+    e.preventDefault();
+    postComment(article_id, newComment, 'tickle122').then((data) => {
+      const fakeComment = {
+        comment_id: 'pending',
+        body: newComment,
+        article_id,
+        author: 'tickle122',
+        created_at: 'Just now',
+        votes: 0,
```

Comment:
We don't **have** to optimistically render comments - this can make it difficult to upvote or downvote the comments, and also makes it hard to delete since it has no comment_id